.github/workflows/ci.yml

```diff
@@ -31,30 +31,115 @@ jobs:
         # file with notes on your contribution.
         git diff --name-only $(git merge-base origin/main HEAD) | grep '^CHANGELOG.md$' && echo "Thanks for helping keep our CHANGELOG up-to-date!"
 
+  style:
+    name: Style
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+
+    - name: Setup Python
+      uses: actions/setup-python@v1
+      with:
+        python-version: 3.8
+
+    - name: Install requirements
+      run: |
+        grep -E '^black' dev-requirements.txt | xargs pip install
+
+    - name: Debug info
+      run: |
+        pip freeze
+
+    - name: Run black
+      if: '! cancelled()'
```

Comment:
Why is this check necessary? Doesn't it always cancel all the checks?

Reply:
This can be removed, it was only need when this check was in the other workflow that had a lot of different steps. And then it survived the copy-paste.